spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -350,38 +350,35 @@ The default Task/Batch event and destination names on the broker are enumerated
 [[spring-cloud-dataflow-launch-tasks-from-stream]]
 == Launching Tasks from a Stream
 
-You can launch a task from a stream by using one of the available `task-launcher` sinks. Currently the only available
-`task-launcher` sink is the `task-launcher-local`, which will launch a task on your local machine.
+You can launch a task from a stream by using one of the available `task-launcher` sinks. Currently the platforms supported
+via the `task-launcher` sinks are
+https://github.com/spring-cloud/spring-cloud-stream-app-starters/tree/master/tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-local[local],
+https://github.com/spring-cloud/spring-cloud-stream-app-starters/tree/master/tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-cloudfoundry[Cloud Foundry], and
+https://github.com/spring-cloud/spring-cloud-stream-app-starters/tree/master/tasklauncher/spring-cloud-starter-stream-sink-tasklauncher-yarn[Yarn].
 
 NOTE: `task-launcher-local` is meant for development purposes only.
 
 A `task-launcher` sink expects a message containing a TaskLaunchRequest object in its payload. From the
 TaskLaunchRequest object the task-launcher will obtain the URI of the artifact to be launched as well as the
 properties and command line arguments to be used by the task.
 
-The `task-launcher-local` can be added to the available sinks by executing the app register command as follows:
-
-```
-app register --name task-launcher-local --type sink --uri maven://org.springframework.cloud.stream.app:task-launcher-local-sink-kafka:jar:1.0.0.BUILD-SNAPSHOT
```

Comment:
Why get rid of registering it?  I was suggesting to have a working example, so including `maven://org.springframework.cloud.stream.app:task-launcher-local-sink-rabbit:1.2.0.RELEASE`